src/app/components/avatar/avatar.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="card col-sm-2" (click)="navigateHero(hero.id, hero.name)">
```

Comment:
dont define how the avatars should be presented on the frame
remove div with row and remove col-sm-2 leaving the rest of the div

Reply:
Fixed, cheers